tests/libcxx/Makefile

```diff
@@ -53,7 +53,7 @@ $(APPDIR):
 $(ROOTFS): run.c
 	$(MYST) mkext2 $(APPDIR) $(ROOTFS)
 
-OPTS += --memory-size=64m
+OPTS += --memory-size=128m
```

Comment:
why was this change necessary? Is it intentional?

Reply:
With 64mb and this code change, these test would fail with memory allocation issue.